biztown/src/components/MultipleChoiceSection.js

```diff
@@ -1,148 +1,172 @@
+          "The amount left over from your paycheck after taxes and deductions is called your:",
+        possibleAnswers: [
+          "Balance",
+          "Gross balance",
+          "Transaction",
+          "Net Pay",
+        ],
+      },
+      {
+        question: "Which one of the following statements is TRUE?",
+        possibleAnswers: [
+          "You can continue to write checks as long as you have checks in your checkbook.",
+          "All checking account transactions should be recorded in the transaction register.",
+          "A direct deposit is when you take your paycheck directly to the bank after you receive it.",
+          "Credit cards and debit cards are the same thing.",
+        ],
+      },
+      {
+        question: "Interests are things people:",
+        possibleAnswers: [
+          "Do well",
+          "Have a natural ability for",
+          "Keep to themselves",
+          "Like to do",
+        ],
+      },
+      {
+        question: "Which two skills are considered soft skills?",
+        possibleAnswers: [
+          "Teamwork, Interests",
+          "Punctuality, Teamwork",
+          "Interests, Compassion",
+          "Dressing appropriately, Compassion",
+        ],
+      },
+      {
+        question: "Which of the following is a business expense?",
+        possibleAnswers: [
+          "Personal care",
+          "Sales",
+          "Revenue",
+          "Advertising",
+        ],
+      },
+      {
+        question:
+          "The revenue left over after paying business costs is called:",
+        possibleAnswers: [
+          "Profit",
+          "Gross balance",
+          "Inventory",
+          "Loss",
+        ],
+      },
+    ];
+    var options = ["a", "b", "c", "d"];
+    var multipleChoiceComponent = multipleChoiceQuestions.map(
+      (questionBlock, index) => {
+        var questionId =
+          index < 9 ? `q0${index + 1}_answer` : `q${index + 1}_answer`;
+        var possibleAnswers = questionBlock.possibleAnswers.map(
+          (possibleAnswer, index) => {
+            var isChecked =
+              questionId in this.props.data &&
+              this.props.data[questionId] === options[index];
             return (
-                <li
-                    className="form-group"
-                    key={questionId}
-                >
-                    <label className="form-question">{questionBlock.question}</label>
-                    <div className="multiple-choice-answer-container">{ possibleAnswers }</div>
-                </li>
+              <div
+                key={options[index]}
+                className="multiple-choice-answer"
+              >
+                <input
+                  className="form-control-radio-btn"
+                  type="radio"
+                  id={questionId}
+                  name={questionId}
+                  value={options[index]}
+                  checked={isChecked}
+                  onChange={this.props.handleChange}
```

Comment:
Unsure where the required checks are happening, here, but maybe it was missed? Looks like this the required doesn't work on chrome. i.e. I am able to go to the next page without any form input. Strangely though, it works swimmingly on Firefox and Safari.

Reply:
Never mind after a hard refresh, seems to work fine.

Reply:
Doubly never mind because it is all in the layout page.